charts/kubernetes-operations/perses-dashboards/prometheus-details.json

```diff
@@ -0,0 +1,1125 @@
+{
+  "kind": "Dashboard",
+  "metadata": {
+    "name": "prometheus-details",
+    "project": "default"
+  },
+  "spec": {
+    "display": {
+      "name": "Prometheus Details"
```

Comment:
Because I just saw it. The Prometheus Plutono dashboard is integrated into the kube-monitoring chart over there.

https://github.com/cloudoperators/greenhouse-extensions/tree/main/kube-monitoring/charts/dashboards